blob/example_openbucket_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package blob_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"gocloud.dev/blob"
+	_ "gocloud.dev/blob/memblob"
+)
+
+func ExampleOpenBucket() {
+
+	// Connect to a bucket using a URL.
+	// This example uses "memblob", the in-memory implementation.
+	// We need to add a blank import line to register the memblob provider's
+	// URLOpener, which implements blob.BucketURLOpener:
+	// import _ "gocloud.dev/blob/memblob"
+	// memblob registers for the "mem" scheme.
+
+	ctx := context.Background()
+	b, err := blob.OpenBucket(ctx, "mem://mybucket")
```

Comment:
Nit: leave off the `mybucket`, it's not needed.

Reply:
Done.